index.d.ts

```diff
@@ -17,7 +21,12 @@ export type navigateTo = (
     component: VueConstructor,
     options?: NavigationEntryVue,
     cb?: () => Page,
-) => Promise<Page>;
+) => Promise<Page>
+
+export type navigateBack = (
+    backstackEntry: BackstackEntry,
+    options?: NavigationEntryVue,
+) => void
```

Comment:
```suggestion
export type navigateBack = (
    options?: NavigationEntryVue,
    backstackEntry?: BackstackEntry,
) => void
```

https://github.com/nativescript-vue/nativescript-vue/blob/f694e346e572f7cc693e22186b929691bdf5c771/platform/nativescript/plugins/navigator-plugin.js#L55